builtin_interfaces/Time stamp

# states
bool is_valid_max_distance_deviation

# values
float64 max_distance_deviation

int64 invalid_count
